api/yorkie/v1/resources.proto

```diff
@@ -109,8 +109,9 @@ message Operation {
     TimeTicket parent_created_at = 1;
     TextNodePos from = 2;
     TextNodePos to = 3;
-    map<string, string> attributes = 4;
-    TimeTicket executed_at = 5;
+    map<string, TimeTicket> created_at_map_by_actor = 4;
+    map<string, string> attributes = 5;
+    TimeTicket executed_at = 6;
```

Comment:
It would be good to add `created_at_map_by_actor` at the end  to preserve protocol's compatibility.

```suggestion
    map<string, string> attributes = 4;
    TimeTicket executed_at = 5;
    map<string, TimeTicket> created_at_map_by_actor = 6; // add
```